CHANGELOG.md

```diff
@@ -1,10 +1,40 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-10-15
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from 1st Dec. 2023. We recommend updating your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
+
+### Release notes 
+
+* Bug Fixes 
+  * Fix for preventing [cilium-operator](https://learn.microsoft.com/azure/aks/azure-cni-powered-by-cilium)  from restarting unmanaged coredns pods
+  * Fix for - [CVE-2023-44487](https://www.cve.org/CVERecord?id=CVE-2023-44487) - The HTTP/2 protocol allows a denial of service (server resource consumption) because request cancellation can reset many streams quickly
```

Comment:
@kevinkrp93 CVE-2023-44487 should point to https://github.com/Azure/AKS/issues/3947